lib/pharos/salt.rb

```diff
@@ -27,6 +27,11 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Call mine.update on salt for every minion specified by targets.
+    def self.mine_update(targets: "*")
```

Comment:
Do we really need an extra method which is a simple alias of the `call` method ?